engine/schema/src/main/java/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -224,6 +225,7 @@ public DatabaseUpgradeChecker() {
                 .next("4.17.2.0", new Upgrade41720to41800())
                 .next("4.18.0.0", new Upgrade41800to41810())
                 .next("4.18.1.0", new Upgrade41810to41900())
+                .next("4.19.0.0", new Upgrade41900to41910())
```

Comment:
Do you intend for this to go in 4.19.1? it is now targetted at main.

Reply:
It goes to the main; just did the changes.